cps/main.c

```diff
@@ -836,11 +836,8 @@ match_bgp4(struct rte_mbuf *pkt, struct gatekeeper_if *iface)
 	if (unlikely(eth_hdr->ether_type != BE_ETHER_TYPE_IPv4))
 		return -ENOENT;
 
-	if (pkt->data_len < minimum_size) {
-		RTE_LOG(NOTICE, GATEKEEPER, "cps: IPv4 BGP packet received is %"PRIx16" bytes but should be at least %hu bytes\n",
-			pkt->data_len, minimum_size);
+	if (pkt->data_len < minimum_size)
 		return -ENOENT;
-	}
 
 	ip4hdr = (struct ipv4_hdr *)&eth_hdr[1];
```

Comment:
I know it's not related to this patch per se, but could you add another patch in this pull request to make the code to support variable Ethernet headers? The other match functions already do it.